src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/__init__.py

```diff
@@ -0,0 +1,12 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import azure.cli.command_modules.cognitiveservices._help  # pylint: disable=unused-import
+
+def load_params(command):
+    import azure.cli.command_modules.cognitiveservices._params
+
+def load_commands():
+    import azure.cli.command_modules.cognitiveservices.commands
+
```

Comment:
Please run `check_style --module cognitiveservices` to fix all the code style issues. This file will fail the pylint because we require 2 blank lines between first level entities.